doc/man7/EVP_RAND-JITTER.pod

```diff
@@ -86,6 +86,12 @@ are additionally conducted.
 L<EVP_RAND(3)>,
 L<EVP_RAND(3)/PARAMETERS>
 
+=head1 HISTORY
```

Comment:
Maybe just a NOTE would be better since the configuration options are not part of EVP_RAND-JITTER.pod

Reply:
Changed.